__init__.py

```diff
@@ -76,6 +131,7 @@ def initialize(self):
             # System events
             self.add_event('system.reboot', self.handle_system_reboot)
             self.add_event('system.shutdown', self.handle_system_shutdown)
+            self.add_event('enclosure.mouth.text', self.handle_show_text)
```

Comment:
I think this is fine for now until we figure out if we want to only do special case for pairing for a while or if we want to support other text messages with this event or do our own enclosure. L129 reset_face only resets the display immediately after first display of showing the pairing code so maybe that is no longer needed? Don't need to mess with now, but I commented it out on my unit so it shows a bit better on Friday.